fcrepo-jms/src/test/java/org/fcrepo/integration/jms/observer/HeadersJMSIT.java

```diff
@@ -131,7 +134,8 @@ public void testIngestion() throws RepositoryException {
         try {
             containerService.findOrCreate(session, testIngested);
             session.save();
-            awaitMessageOrFail(testIngested, NODE_ADDED_EVENT_TYPE, null);
+            final Message ingestMsg = awaitMessageOrFail(testIngested, NODE_ADDED_EVENT_TYPE);
+            assertTrue(getResourceTypes(ingestMsg).contains("fedora:Resource"));
```

Comment:
ajs6f++

IMO the JMS headers that we're arguing about shouldn't even be part of the SPI specification but rather a "handy extra feature of the JMS implementation of that spec" -- if they are even part of that implementation at all. The key thing is to have an RDF serialization so the type of ambiguity I'm trying to avoid is resolved properly.